### PR TITLE
Init commit

### PR DESCRIPTION
**Pull request by AnnRiera**

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+coverage/
+.circleci/
+*.tsbuildinfo
+*.js.map
+src/
+.eslintrc.js
+.gitignore
+.prettierrc
+.travis.yml
+nest-cli.json
```
Comment (jarcodallo) [inline .npmignore:12]: We don't need `nest-cli.json` here.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    "build-esnext": "tsc -p tsconfig.json --outDir lib.esnext -t es2015 -m commonjs",
+    "build-esm": "tsc -p tsconfig.json --outDir lib.esm -t es2015 -m es2015",
+    "lint:ci": "eslint \"{src,apps,libs,test}/**/*.ts\"",
+    "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/blockcoders/harmony-marketplace-sdk.git"
+  },
+  "keywords": [
```
Comment (jarcodallo) [inline package.json:34]: Add nft, hr721, hr1155, harmony-sdk and one.

File: .npmignore (hunk 1 above)
Comment (AnnRiera) [reply .npmignore:12]: Got it!

File: package.json (hunk 2 above)
Comment (AnnRiera) [reply package.json:34]: Done!

**Review by jarcodallo — APPROVED** [review]

**Pull request merged**